satellite/admin/project_test.go

```diff
@@ -144,6 +145,45 @@ func TestAddProject(t *testing.T) {
 	})
 }
 
+func TestDeleteProject(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount:   1,
+		StorageNodeCount: 0,
+		UplinkCount:      1,
+		Reconfigure: testplanet.Reconfigure{
+			Satellite: func(log *zap.Logger, index int, config *satellite.Config) {
+				config.Admin.Address = "127.0.0.1:0"
+			},
+		},
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		address := planet.Satellites[0].Admin.Admin.Listener.Addr()
+		projectID := planet.Uplinks[0].Projects[0].ID
+
+		apikeys, err := planet.Satellites[0].DB.Console().APIKeys().GetPagedByProjectID(ctx, projectID, console.APIKeyCursor{
+			Page:   1,
+			Limit:  10,
+			Search: "",
+		})
+		require.NoError(t, err)
+		require.Len(t, apikeys.APIKeys, 1)
```

Comment:
How about if you check that deleting the project at this point will fail with the expected error code?

Reply:
I thats a good idea. Lemme add that.

Reply:
Added the additional validation!